deployer.py

```
@@ -79,34 +79,42 @@ def _deploy(self, images_to_replace, dependant_services):
         self.compose.run_compose(['-c', 'docker-compose run --rm api ./conversion_service/manage.py shell < create_service_user_noinput.py'], run_plain_bash=True, extra_environment_variables=extra_env)
         self.compose.run_compose(['stop'] + dependant_services, extra_environment_variables=extra_env)
 
-    def dry_run(self, containers, dependant_services):
+    def recreate_osmaxx_starter(self, containers, starter_container_name):
+        try:
+            subprocess.check_call('docker stop {0}'.format(starter_container_name).split(' '))
+        except FileNotFoundError:  # skip if stopping the `starter_container_name` fails
```

If I understood you correctly, the error will only be silently discarded if stopping the container failed because there is no (running?) container of that name, and if stopping failed for _any_ other reason, the error will still be raised.